rust/src/transformations/make_stable_expr/expr_col/mod.rs

```diff
@@ -0,0 +1,115 @@
+    (ExprDomain, M): MetricSpace,
+{
+    let Expr::Column(col_name) = expr else {
+        return fallible!(MakeTransformation, "Expected Column expression");
+    };
+    let col_name = col_name.to_string();
+
+    let mut output_domain = input_domain.clone();
+    output_domain
+        .frame_domain
+        .series_domains
+        .retain(|v| v.field.name == col_name);
+
+    // ensure that column exists in the domain
+    output_domain.active_series()?;
+
+    Transformation::new(
+        input_domain,
+        output_domain,
+        Function::new_fallible(
+            // in most other situations, we would use `Function::new_expr`, but we need to return a Fallible here
+            move |(plan, expr): &(LogicalPlan, Expr)| -> Fallible<(LogicalPlan, Expr)> {
+                if expr != &Expr::Wildcard {
+                    return fallible!(
+                        FailedFunction,
+                        "make_col has to be the first expression in the expression chain"
+                    );
+                }
+                Ok((plan.clone(), col(&*col_name)))
+            },
+        ),
+        input_metric.clone(),
+        input_metric,
+        StabilityMap::new(Clone::clone),
+    )
+}
+
+#[cfg(test)]
+mod test_make_col {
+    use crate::metrics::SymmetricDistance;
+    use crate::transformations::{polars_test::get_test_data, StableExpr};
+
+    use super::*;
+
+    #[test]
+    fn test_make_col_expr() -> Fallible<()> {
+        let (lf_domain, lf) = get_test_data()?;
+        let expr_domain = lf_domain.select();
+        let expr_exp = col("A");
+        let t_col: Transformation<_, _, _, SymmetricDistance> = expr_exp
+            .clone()
+            .make_stable(expr_domain.clone(), SymmetricDistance)?;
+        let expr_res = t_col.invoke(&(lf.logical_plan, all()))?.1;
+
+        assert_eq!(expr_res, expr_exp);
+
+        Ok(())
+    }
+
+    #[test]
+    fn test_make_col_expr_no_wildcard() -> Fallible<()> {
+        let (lf_domain, lf) = get_test_data()?;
+        let expr_domain = lf_domain.select();
+
+        let expr_exp = col("A");
+        let t_col: Transformation<_, _, _, SymmetricDistance> =
+            expr_exp.make_stable(expr_domain.clone(), SymmetricDistance)?;
+        let error_res = t_col
+            .invoke(&(lf.logical_plan, col("B")))
+            .map(|v| v.1)
+            .unwrap_err()
+            .variant;
+        assert_eq!(error_res, ErrorVariant::FailedFunction);
+
+        Ok(())
+    }
+
+    #[test]
+    fn test_make_col_expr_wrong_col() -> Fallible<()> {
+        let (lf_domain, _) = get_test_data()?;
+        let expr_domain = lf_domain.select();
+
+        let expr_exp = col("nonexistent");
+        let error_res = expr_exp
+            .make_stable(expr_domain.clone(), SymmetricDistance)
+            .map(|_: Transformation<_, _, _, SymmetricDistance>| ())
+            .unwrap_err()
+            .variant;
+
+        assert_eq!(error_res, ErrorVariant::FailedFunction);
```

Comment:
Do we get a different error message here and above? Consider adding a message check to the test, or make finer-grained error classes, if we don't want to compare strings.